lib/active_shipping/carriers/ups.rb

```diff
@@ -383,7 +384,7 @@ def build_shipment_request(origin, destination, packages, options={})
               end
             end
 
-            if options[:origin_account]
+            if options[:negotiated_rates]
```

Comment:
This is a breaking change. This gem will need to release a major version if it is following SemVer. Or maybe just make this backwards compatible.